library/package.json

```diff
@@ -3,7 +3,7 @@
   "name": "@kaliber/build",
   "description": "Zero configuration, opinionated webpack / react build setup",
   "scripts": {
-    "lint": "eslint -c .eslintrc ."
+    "lint": "eslint -c .eslintrc.js ."
```

Comment:
do we actually need the `-c .eslintrc.js` here? I am under the impression that it will pick it up anyway.